build/yaml/samples/named-port-sample/rule-in-policy-applied-to.yml

```diff
@@ -0,0 +1,61 @@
+#  test: kubectl exec -ti client -n client -- curl web ip
```

Comment:
typo in commit title and PR title! And in the commit message, I think the word should be "scenarios", not "sceneries".